client/components/Place/Place.js

```diff
@@ -2,24 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './Place.css';
 
-const Place = ({ place }) => {
+const Place = ({ place: { data } }) => {
+  const { name, address, phone, categories, price, reviewCount, rating } =
+    data || {};
```

Comment:
Ohh yes i added it to have error and fetching meta data. Would that reducer be inside of placeReducer since I wanted place to have its own error and fetching state.

Reply:
i was thinking more like a asyncall reducer specifically targeting async actions

Reply:
But that would not be specific to that entity (e.g. place)?

Reply:
as you have defined type for existing calls like REQUEST[SUCESS] probably you can do something similar PLACE[BLA]

Reply:
I don't see how adding another action type for fetch place will help in this scenario. I don't seem to understand the problem of tying the fetching and error status to the place itself. Can you please expound more?